cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -92,16 +92,14 @@ void run_mnist(const std::string& model_name,
       for (int i=0; i<num_devices; i++ ) {
```

Comment:
To note: I notice that here the .cpp file is named 'inference_scaling_imagenet.cpp' whereas in throughput it is named 'throughput_scaling.cpp', I would suggest changing to 'throughput_scaling_imagenet.cpp'

Reply:
Tbh this might be something I can include in my ticket, #28 , let me know if you agree

Reply:
Actually the throughput does not use ImageNet, so I think it would be a bit misleading to name it that way.

Reply:
oop that would be mega misleading